packages/contracts-bedrock/snapshots/state-diff/Kontrol-Deploy.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
This will fall out of date quickly - must this be regenerated each after each change?

Reply:
The file is being tracked as per this comment https://github.com/ethereum-optimism/optimism/pull/8634#discussion_r1430471146.
However, it is true that the file will change when some contents of the contracts deployed in [KontrolDeployment](https://github.com/runtimeverification/_audits_Ethereum-optimism_pausability/blob/kontrol-proofs/packages/contracts-bedrock/test/kontrol/KontrolDeployment.sol) are changed, but only contents related to deployment. The `Kontrol-Deploy.json` will only change if:
- The initialization logic of the contracts in the `KontrolDeployment` changes, or
- The functions in the deployment script change

So, it only tracks deployment-related changes. Changing anything else in those contracts should not change `Kontrol-Deploy.json`

Reply:
Resolving this. Feel free to unresolve if there are any further concerns